README.md

```diff
@@ -14,19 +14,19 @@ This project requires a [Docker](https://www.docker.com) installation.
 ## Installation and Execution ##
 
 - Build the docker image:
-  - `docker-compose build`
+  - `docker compose build`
 - Change the credentials in the following configuration files:
   - `secrets/admiral.yml`
   - `secrets/redis.conf`
-  - `docker-compose.yml`
+  - `compose.yaml`
 - Start the composition:
-  - `docker-compose up`
+  - `docker compose up`
   - alternately it can be started in [swarm mode](https://docs.docker.com/engine/swarm/):
-  `docker stack deploy admiral --compose-file docker-compose.yml`
+  `docker stack deploy admiral --compose-file compose.yaml`
 - Monitor the system:
   - [http://localhost:5555](http://localhost:5555)
 - Optional: Run the code tests
-  - `docker-compose -f docker-compose-dev.yml run test`
+  - `docker compose -f docker-compose-dev.yml run test`
```

Comment:
`docker-compose-dev.yml` should be renamed `compose-dev.yaml` as part of this PR and the README should be updated accordingly.  Also, the same changes made to `compose.yaml` file should be applied to `compose-dev.yaml`.

Reply:
See [05e43a4](https://github.com/cisagov/admiral/pull/79/commits/05e43a488ca78fbbc9aec78d5d5f097ad0bfc09d), [ecee5bf](https://github.com/cisagov/admiral/pull/79/commits/ecee5bf705f33ad0b1e7051d71b1b647f30819dc), and [6bed5ac](https://github.com/cisagov/admiral/pull/79/commits/6bed5ac774930eb16ec0ce51db5cbb607dc8d38f).